pkg/allowedpaths/cmd_darwin.go

```diff
@@ -0,0 +1,96 @@
+//go:build darwin
+// +build darwin
+
+package allowedpaths
+
+var knownPaths = map[string]map[string]bool{
```

Comment:
One callout here... This will _require_ we define per platform. There's no shared mechanism. I think that's okay, but I did want to note it.

Reply:
I _think_ that's my preference -- I figured since we're switching from hardcoded paths, we'd want the narrowest possible definitions, and per-platform gives us that. But since we have to do a prefix allowlist (at least for linux) anyway, maybe it doesn't matter.

Reply:
We do share some posix things, but I suspect it's just not common enough to matter. Most of the mac execs are really mac specific.

Dunno though, I think with our slack conversation we'll have more rope